src/api/init.ts

```diff
@@ -99,6 +100,7 @@ export const StacksApiRoutes: FastifyPluginAsync<
     async fastify => {
       await fastify.register(BlockRoutesV2, { prefix: '/blocks' });
       await fastify.register(BurnBlockRoutesV2, { prefix: '/burn-blocks' });
+      await fastify.register(TenureHeightRoutesV2, { prefix: '/tenure-height' });
```

Comment:
For v2 endpoints we're trying to keep route names as REST-ful as possible, so we want to group endpoints according to the entity that they return. Adding `tenure-height` here would not fit the standard.

Could you instead add a `tenure_height` query param to the `/extended/v2/blocks` endpoint? That will also allow you to get cursor pagination for free and will work much better with API clients.